server/templates/base_page.html

```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script async>
```

Comment:
Yeah, not sure why I added the `async` tag...

However, @jlfwong I don't think the r.js optimizer is being used. If you look in the [deploy](https://github.com/UWFlow/rmc/blob/master/deploy.sh#L29) file you'll see that part is commented out. So if we do want to re-enable that, I'd prefer to do it in a separate commit, and keep the way I set the environment in the PR, as I've tested it locally. 

@divad12 let me know what you think as well, since I think the React dev/prod loading issue is the only blocking one right now, aside from a few nits you pointed out.